packages/terra-show-hide/src/ShowHide.module.scss

```diff
@@ -0,0 +1,17 @@
+:local {
+  .show-hide {
+    display: flex;
```

Comment:
How come this container needs the flex display? All I see in there is a single button.

Reply:
I saw that you were putting the button in the middle, left or right depending on the prop. I'm curious what others think. I feel we should just use the text-align property instead.

Reply:
I don't know enough about CSS to know the best strategy. @StephenEsser helped me with this part. I tested it in IE10 and it works fine. Also works when switching between for LTR and RTL